doc/sphinx/compiling.rst

```diff
@@ -5,21 +5,22 @@ Cantera Compilation Guide
 *************************
 
 .. toctree::
-   :hidden:
 
    SCons Configuration Options <configuring>
 
 This guide contains instructions for compiling Cantera on the following
 operating systems:
 
-* Linux
+* `Linux`_
 
-  * Ubuntu 12.04 LTS (Precise Pangolin) or newer; 16.04 LTS (Xenial Xerus) or
-    newer is recommended
-  * Debian 7.0 (Wheezy) or newer; 8.0 (Jessie) or newer is recommended
-  * Fedora 20 or newer; 22 or newer is recommended
+  * :ref:`sec-ubuntu-debian-reqs`
+  * :ref:`sec-fedora-reqs`
+  * :ref:`sec-opensuse-reqs`
+
+* `Windows`_
+
+  * :ref:`sec-windows-reqs`
 
-* Windows 7 or newer (32-bit or 64-bit versions)
 * OS X 10.9 (Mavericks) or newer; 10.10 (Yosemite) or newer is recommended
```

Comment:
OK, judging from the apple.stackexchange post and the experience with the broken Numpy that Apple provides, I'm inclined to believe that the system-installed Python isn't really there for the user, but really just for use by Apple, and that any developer would be better off using Homebrew Python.